recipes/vtk-m/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "vtk-m" %}
+{% set version_major = "2" %}
+{% set version_minor = "0" %}
+{% set version_patch = "0" %}
+{% set version = version_major + "." + version_minor + "." + version_patch %}
+{% set common_flags = "-DCMAKE_BUILD_TYPE=Release -DVTKm_ENABLE_EXAMPLES=ON -DVTKm_INSTALL_EXAMPLES=OFF" %}
```

Comment:
We have a bot which can create pull request to automatically update package, but I'm not sure it works when you concatenate the version variable this way. I suggest that you do the opposite direction and split the version into parts.

```yaml
{% set version = "2.0.0" %}
{% set version_major = {{ version }}.split('.')[0] %}
{% set version_minor = {{ version }}.split('.')[1] %}
```